randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -133,16 +131,21 @@ Extra - asset_id: 70
           Speed Booster
           # https://www.youtube.com/watch?v=7bHxm5TttRI
           Morph Ball and Machball (Intermediate)
+          # https://youtu.be/stNcGyyLCB0
+          Movement (Expert)
   > Pickup (Missile Expansion)
```

Comment:
cant put line at room, but its worth adding a comment that VARIA supports more patches for this one.

Reply:
What do you mean?

Reply:
![grafik](https://github.com/randovania/randovania/assets/38186597/22ec2d70-d105-44f0-8653-e14e25bc7d75)

Altho now that i think about it, unsure if it has any logical implications